cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -91,6 +91,39 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
           });
         });
 
+        if (service === 'news') {
+          it('should render a visible guidance message', () => {
+            cy.window().then(win => {
+              const aresMedia = getBlockData(
+                'video',
+                win.SIMORGH_DATA.pageData,
+              );
+              const longGuidanceWarning =
+                aresMedia.model.blocks[1].model.blocks[0].model.versions[0]
+                  .warnings.long;
+
+              cy.get('div[class^="StyledVideoContainer"]')
+                .eq(0)
+                .within(() => {
+                  // Check for video with guidance message
+                  if (longGuidanceWarning) {
```

Comment:
If this test should only run when Guidance data exists, why does it begin without checking this line first? If there is a requirement to _also_ check that this component _is not visible_, then we should update the parent ticket and this test's title.

Reply:
No, it doesn't only run when when Guidance exists, it also checks when guidance is not available

Reply:
I understand, but do we need to check that it is not visible if it doesn't exist in the block data? We don't do this anywhere else in our suite.

Reply:
We use it for Useful-Links to make sure that when at a certain breakpoint the component isn't visible.

Reply:
Thanks, @jamesbrumpton, that makes sense in Useful-Links' case. In Guidance's case, why do we need to check that it _is not visible_  if there isn't a video block in `pageData`?

Reply:
My understanding of how Ray & I have written the test is that *if* we find `warnings.long` within `version[0]` we should check that there's a visible component. However, if there's no presence of this within the data then the guidance component should not be visible. Unless I've misunderstood? This test shouldn't be running at all if there's no `video` presence within `pageData`.

Reply:
> This test shouldn't be running at all if there's no video presence within pageData.

- This test starts running on L105 before checking whether there is a video block, and
- It shouldn't need to do anything if the page doesn't contain a video block, but it continues to check whether Guidance _does not exist_ within a contain that won't exist.

P.S. I'm happy to do a call to discuss this further.